sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdkBuilder.java

```diff
@@ -527,7 +527,7 @@ public AutoConfiguredOpenTelemetrySdk build() {
 
   @Nullable
   private static AutoConfiguredOpenTelemetrySdk maybeConfigureFromFile(ConfigProperties config) {
-    String configurationFile = config.getString("otel.config.file");
+    String configurationFile = config.getString("otel.experimental.config.file");
```

Comment:
This PR suggests changing to `otel.experimental.config.file` without a deprecation cycle for `otel.config.file`. Alternatively, we could:
- Deprecate `otel.config.file` and warn if its used
- Retain support for `otel.config.file` permanently, but only document `otel.experimental.config.file`. Essentially, `otel.config.file` is an undocumented alias for `otel.experimental.config.file`, anticipating that in the future when file config is stable, `otel.config.file` will be the name we land on.

Reply:
How about removing support for it, but log the warning if it appears to be set?

Reply:
> How about removing support for it, but log the warning if it appears to be set?

Sounds good to me.